### PR TITLE
Add extra params

### PR DESCRIPTION
**Pull request by GijsvanDulmen**
# Changes

Add an option to pass extra parameters to helm.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [X] Follows the [authoring recommendations](https://github.com/tektoncd/catalog/blob/main/recommendations.md)
- [X] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [X] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality of task changed or new task added)
- [X] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [X] Complies with [Catalog Organization TEP][TEP], see [example]. **Note** [An issue has been filed to automate this validation][validation]
  - [X] File path follows  `<kind>/<name>/<version>/name.yaml`
  - [X] Has `README.md` at `<kind>/<name>/<version>/README.md`
  - [X] Has mandatory `metadata.labels` - `app.kubernetes.io/version` the same as the `<version>` of the resource
  - [X] Has mandatory `metadata.annotations` `tekton.dev/pipelines.minVersion`
  - [X] mandatory `spec.description` follows the convention

          ```

          spec:
            description: >-
              one line summary of the resource

              Paragraph(s) to describe the resource.
          ```

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md)
for more details._

---

[TEP]: https://github.com/tektoncd/community/blob/master/teps/0003-tekton-catalog-organization.md
[example]: https://github.com/tektoncd/catalog/tree/master/task/git-clone/0.1
[validation]:  https://github.com/tektoncd/catalog/issues/413

### CODE REVIEW

File: task/helm-upgrade-from-source/0.3/helm-upgrade-from-source.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  steps:
+    - name: upgrade
+      image: $(params.helm_image)
+      workingDir: /workspace/source
+      script: |
+        echo current installed helm releases
+        helm list --namespace "$(params.release_namespace)"
+
+        echo installing helm chart...
+        helm upgrade --install --wait --values "$(params.charts_dir)/$(params.values_file)" --namespace "$(params.release_namespace)" --version $(params.release_version) $(params.release_name) $(params.charts_dir) --debug --set "$(params.overwrite_values)" $(params.upgrade_extra_params)
```
Comment (fhopfensperger) [inline task/helm-upgrade-from-source/0.3/helm-upgrade-from-source.yaml:52]: Could you please quote the Parameters to satisfy the linter?

Comment (GijsvanDulmen) [reply task/helm-upgrade-from-source/0.3/helm-upgrade-from-source.yaml]: Done! I however can't quote the last parameter due to it's usage.

**Review by fhopfensperger — APPROVED** [review]

**Pull request merged**